lib/fortnox/api/models/row/entity.rb

```diff
@@ -0,0 +1,66 @@
+require "fortnox/api/models/entity/base"
+
+module Fortnox
+  module API
+    module Entities
+      class Row < Fortnox::API::Entities::Base
```

Comment:
No, it doesn't. It cannot be updated or saved since it is not a Fortnox resource, so it should not be an entity at all. The question is then where to put this model. `Currency` and `CountryCode` is placed in `fortnox/api/attributes/`. I would say that `Row` is more than an attribute...